modules/nickserv.cpp

```diff
@@ -113,7 +133,9 @@ class CNickServ : public CModule {
              sMessage.StripControls_n().find(
                  "type /NickServ IDENTIFY password") != CString::npos ||
              sMessage.StripControls_n().find(
-                 "type /msg NickServ IDENTIFY password") != CString::npos) &&
+                 "type /msg NickServ IDENTIFY password") != CString::npos ||
+             sMessage.StripControls_n().find(
+                 GetNV("CustomTrigger")) != CString::npos) &&
```

Comment:
What if `GetNV("CustomTrigger")` is an empty string? This would always match, wouldn't it?

Reply:
Actually, I just made the if statement even more ridiculous to fix this. :joy_cat: See cc523d4…